src/app/api/services/dot-license/dot-license.service.ts

```diff
@@ -27,6 +76,28 @@ export class DotLicenseService {
         return this.getLicense().pipe(map((license) => license['level'] >= 200));
     }
 
+    canAccessEnterprisePortlet(url: string): Observable<boolean> {
+        return this.isEnterprise().pipe(
+            take(1),
+            map((isEnterpriseUser: boolean) => {
+                const urlMatch = this.checksIfEnterpriseUrl(url);
+                return urlMatch ? urlMatch && isEnterpriseUser : true;
+            })
+        );
+    }
+
+    private checksIfEnterpriseUrl(url: string): boolean {
+        let urlMatch = false;
```

Comment:
`return enterprisePorlets.map(({url}) => url)).includes(url)`

Reply:
not possible, because I need to set the Data of the specific portlet to be displayed on the Not Licensed component
https://github.com/dotCMS/core-web/blob/e4cea2194bf1988f9b03fd6c2fa9299e96990339/src/app/api/services/dot-license/dot-license.service.ts#L94

Reply:
I changed with a filter, instead of a FOR